examples/README.md

```diff
@@ -0,0 +1,20 @@
+## Demo Notebook
+
+We can launch a jupyter notebook to try out the library or transforms or to build a usecase.
+
+
+## One time Step
+
+1. Run this step install the dependancies. 
+
+**Prepare virtualenv**
+
+`make venv`
+
+## Launch
+
+2. After dependancies are installed, we can launch the jupyter notebook
+
+`make jupyter`
+
```

Comment:
Can we  suggest a zip file for use? Petros used our repo.  Maybe that  can work?

Reply:
Yeah, that is a good starting point till we get data approved.